modyn/supervisor/internal/grpc_handler.py

```diff
@@ -276,69 +278,47 @@ def start_training(
             optimizer_config["source"] = optimizer["source"]
             optimizer_config["param_groups"] = []
             for param_group in optimizer["param_groups"]:
-                config_dict = param_group["config"] if "config" in param_group else {}
+                config_dict = param_group.get("config") or {}
                 optimizer_config["param_groups"].append({"module": param_group["module"], "config": config_dict})
             optimizers_config[optimizer["name"]] = optimizer_config
 
         lr_scheduler_configs = {}
-        if "lr_scheduler" in pipeline_config["training"]:
+        if pipeline_config["training"].get("lr_scheduler"):
             lr_scheduler_configs = pipeline_config["training"]["lr_scheduler"]
-            if "config" not in lr_scheduler_configs:
-                lr_scheduler_configs["config"] = {}
+            lr_scheduler_configs["config"] = lr_scheduler_configs.get("config", None)
 
-        if "config" in pipeline_config["training"]["optimization_criterion"]:
-            criterion_config = json.dumps(pipeline_config["training"]["optimization_criterion"]["config"])
-        else:
-            criterion_config = "{}"
+        criterion_config = json.dumps(pipeline_config["training"]["optimization_criterion"].get("config") or {})
 
-        if "epochs_per_trigger" in pipeline_config["training"]:
-            epochs_per_trigger = pipeline_config["training"]["epochs_per_trigger"]
-        else:
-            epochs_per_trigger = 1
+        epochs_per_trigger = pipeline_config["training"].get("epochs_per_trigger") or 1
```

Comment:
above can both be re-written with the second arg of `get`

Reply:
No